<?xml version="1.0"?>
<launch>

  <node pkg="mcr_linear_regression" type="laser_scan_linear_regression_node" name="linear_regression" ns="mcr_perception" respawn="false" output="screen" >
        <param name="scan_topic" value="/scan_front" />
  </node>

  <node pkg="mir_base_placement" type="base_placement_node" name="base_placement" ns="mir_navigation" respawn="false" output="screen">
  	<param name="cmd_vel_topic" value="/cmd_vel_prio_low" />
  	<param name="linear_regression_service" value="/scan_front_linearregression" />
  	<param name="laser_axis" value="+x" />
  	<param name="max_linear_velocity" value="0.18" />
  	<param name="max_angular_velocity" value="0.18" />
  	<param name="lin_p" value="0.5" />
  	<param name="ang_p" value="1.3" />
  	<param name="lin_d" value="0.05" />
  	<param name="ang_d" value="0.05" />
    <param name="translation_error_tolerance" value="0.04" />
    <param name="angular_error_tolerance" value="0.02" />
  </node>

</launch>
